net-im/swiften/swiften-4.0.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit scons-utils toolchain-funcs
+
+DESCRIPTION="A C++ library for implementing XMPP applications"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/swift-${PV}/swift-${PV}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="avahi debug doc expat experimental icu idn upnp"
+
+DEPEND="dev-db/sqlite:3
+	dev-libs/boost:=
+	dev-libs/openssl:0=
+	sys-libs/zlib
+	avahi? ( net-dns/avahi )
+	doc? ( app-text/docbook-xsl-stylesheets
+		app-text/docbook-xml-dtd:4.5 )
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2 )
+	icu? ( dev-libs/icu )
+	idn? ( net-dns/libidn )
+	upnp? (	net-libs/libnatpmp
+		net-libs/miniupnpc )"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/swift-${PV}"
+
+src_prepare() {
+	default
+
+	# Remove those directories, as the system libs are used
+	rm -fr 3rdParty/Boost 3rdParty/DocBook 3rdParty/Expat 3rdParty/LibIDN 3rdParty/LibMiniUPnPc 3rdParty/LibNATPMP 3rdParty/Lua 3rdParty/OpenSSL 3rdParty/SCons 3rdParty/SQLite 3rdParty/Unbound 3rdParty/Zlib || die
+
+	# Remove 'Limber', 'Slimber' and 'Sluift', as compiling headless
+	rm -fr Limber Slimber Sluift || die
+}
+
+src_configure() {
+	MYSCONS=(
+		ar="$(tc-getAR)"
+		allow_warnings="$(usex debug yes no)"
```

Comment:
this toggles "-Werror" and thus should be always "yes"

Reply:
Thanks, fixed.